README.md

```diff
@@ -46,14 +49,14 @@ $ yarn workspace api start
 $ yarn workspace app serve
 ```
 
-## To build:
+### To build extension:
 
-- Extension:
 ```bash
+$ export LUNIE_API=https://production-api.lunie.io
 $ yarn workspace extension build
 ```
 
-## To build enabling localhost connection
+### To build extension enabling localhost connection
 
 ```bash
 $ yarn run build:dev
```

Comment:
```suggestion
$ export LUNIE_API=https://staging-api.lunie.io
$ yarn workspace extension build:dev
```

Reply:
I remember adding it and then deleting it! 🤦 

Thanks